server/service/individualServices/AuthorizationApplication.js

```diff
@@ -218,13 +215,15 @@ exports.isOpeartionisExistAsync = async function (applicationName, applicationRe
     }
 }
 
-exports.IsApplicationExists = async function (applicationaName, ReleaseNumber, authorization) {
+exports.IsApplicationExists = async function (applicationaName, releaseNumber,authorization) {
```

Comment:
change method name to isApplicationExists. change the typo in the argument "applicationaName" to "applicationName"